backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -246,6 +255,7 @@ public void enableNativeIntegration() {
 
     /**
      * Capture unhandled native exceptions (Backtrace database integration is required to enable this feature).
+     *
```

Comment:
empty comment.

Reply:
The Android Studio formatting adds an extra newline between comment body and params. Should we keep Android Studio formatting @konraddysput ?

Reply:
It's fine if we will be consistent in the rest comments.